README.md

```diff
@@ -1,5 +1,6 @@
 # .NET Aspire
 
+[![Tests](https://github.com/dotnet/aspire/actions/workflows/tests.yml/badge.svg?branch=main&event=push)](https://github.com/dotnet/aspire/actions/workflows/tests.yml)
```

Comment:
I'd call this the "Build Status" as well. We could consider calling this "Build Status (GitHub)" and the next one "Build Status (AzDO)".